tasks.py

```diff
@@ -31,54 +29,23 @@ def _check_if_directory_in_path(pth, target):
     return False
 
 
-try:
-    from nbformat import read
-except ImportError:
-    echo("Warning: nbformat could not be imported, some tasks may not work")
-
-
-@task
-def docs(ctx):
+def docs(args):
+    del args  # unused
     if not WINDOWS:
-        run(ctx, 'py.test --nbval-lax --current-env nbgrader/docs/source/user_guide/*.ipynb')
-    run(ctx, 'python nbgrader/docs/source/build_docs.py')
-    run(ctx, 'python nbgrader/docs/source/clear_docs.py')
-    run(ctx, 'make -C nbgrader/docs html')
-    run(ctx, 'make -C nbgrader/docs linkcheck')
+        run('py.test --nbval-lax --current-env nbgrader/docs/source/user_guide/*.ipynb')
+    run('python nbgrader/docs/source/build_docs.py')
+    run('python nbgrader/docs/source/clear_docs.py')
+    run('make -C nbgrader/docs html')
+    run('make -C nbgrader/docs linkcheck')
 
 
-@task
-def cleandocs(ctx):
-    run(ctx, 'python nbgrader/docs/source/clear_docs.py')
+def cleandocs(args):
+    del args  # unused
+    run('python nbgrader/docs/source/clear_docs.py')
 
 
-def _run_tests(ctx, mark=None, skip=None, junitxml=None):
-    if not WINDOWS:
-        import distutils.sysconfig
-        site = distutils.sysconfig.get_python_lib()
-        sitecustomize_path = os.path.join(site, "sitecustomize.py")
-        if os.path.exists(sitecustomize_path):
-            with open(sitecustomize_path, "r") as fh:
-                sitecustomize = fh.read()
-            with open(sitecustomize_path, "w") as fh:
-                fh.write(re.sub(
-                    "^### begin nbgrader changes$.*^### end nbgrader changes$[\n]",
-                    "",
-                    sitecustomize,
-                    flags=re.MULTILINE | re.DOTALL))
-
-        with open(sitecustomize_path, "a") as fh:
-            fh.write(dedent(
-                """
-                ### begin nbgrader changes
-                import coverage; coverage.process_startup()
-                ### end nbgrader changes
-                """
-            ).lstrip())
-
+def _run_tests(mark, skip, junitxml):
     cmd = []
-    if not WINDOWS:
```

Comment:
Not directly related to this PR, but pytest now prefers `pytest` for command execution. Since we have a dependency on a recent version of pytest, we should update: `cmd.append('py.test')`

Reply:
Ah, I didn't realize that, thanks! I went ahead and made those changes in this PR as well.